id_test.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const fl = require('.');
```

Comment:
Could you change the path to `'./'`? `'.'` has always worked for me, but I've seen it fail for other people.